exporter/elasticsearchexporter/model.go

```diff
@@ -1074,3 +1079,11 @@ func mergeGeolocation(attributes pcommon.Map) {
 		}
 	}
 }
+
+func safeUint64ToInt64(v uint64) int64 {
```

Comment:
This func is appearing the 2nd time. Is it intentional to avoid sharing a dependent util file?

Reply:
I can do that, but I didn't see any "generic util package", and didn't want to introduce this kind of smell. Especially for such a simple logic.

Reply:
i'm fine with this duplication. Even better if we can get rid of it after adding Uint and friends